src/components/Dashboard.tsx

```diff
@@ -58,7 +59,7 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
       ? USER_ROLES.APPLICANT
       : USER_ROLES.REVIEWER
 
-  const [loadingFilters, setLoadingFilters] = useState(false)
+  const [loadingFilters, setLoadingFilters] = useState<number>(0)
```

Comment:
FYI -- when initialising a variable, if it's type can be inferred, you don't need to specify it.
So: `const [loadingFilters, setLoadingFilters] = useState(0)` is fine here. Typescript will enforce it as `number` elsewhere.

Reply:
Also, maybe name it to something a little more specific -- `loadedFiltersCount` perhaps?

If you're not already familiar with it, the VSCode command "Rename symbol" (right-click menu) is super-useful for renaming variables.

Reply:
That's great, thanks Carl. I very much appreciate the tips and tricks - I hadn't heard of that one before.